scripts/tools/tools-schema.json

```diff
@@ -26,26 +26,22 @@
             "properties": {
                 "websiteUrl": {
                     "type": "string",
-                    "description": "You can provide URL to the website where your project hosts some demo or project landing page.",
-                    "format": "uri"
+                    "description": "You can provide URL to the website where your project hosts some demo or project landing page."
                 },
                 "docsUrl": {
                     "type": "string",
-                    "description": "You can provide URL to project documentation in case you have more than just a readme file.",
-                    "format": "uri"
+                    "description": "You can provide URL to project documentation in case you have more than just a readme file."
                 },
                "repoUrl": {
                   "type": "string",
-                  "description": "You can provide URL to project codebase in case you have more than one tool present inside single repository.",
-                  "format": "uri"
+                  "description": "You can provide URL to project codebase in case you have more than one tool present inside single repository."
                 }
             }
         },
         "filters": {
             "type": "object",
             "additionalProperties": false,
             "required": [
-                "technology",
```

Comment:
I wonder if we should have the `unknown` technology for such a case 🤔

Reply:
What do you mean by `unknown` technology?

Reply:
It's not specified/hard to say.

Reply:
But if a certain tool isn't using any technology, it can left this field in the file. We are handling a case where user doesn't specify any technology in the tool or even specified any technology not mentioned in our list, it can be also be handled in script by providing a dynamic tag to it.

Reply:
yeah, leave it as not required for now